tls-tester/pom.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-  <modelVersion>4.0.0</modelVersion>
```

Comment:
Is this the only version I need to raise ? @ivy-rew

Reply:
modelVersion must remain 4.0.0
but the version of the artifact must be raised from 8.0.0V > 10.0.0-SNAPSHOT